tests/test_ansi2html.py

```diff
@@ -107,10 +107,10 @@ def test_conversion_as_command(
         with open(join(_here, "ansicolor.html"), "rb") as output:
             expected_data = "".join(read_to_unicode(output))
 
-        def f() -> StringIO:
-            return StringIO(test_data)
-
-        with patch("sys.stdin", new_callable=f):
+        with patch(
+            "sys.stdin",
+            TextIOWrapper(BytesIO(test_data.encode("utf-8"))),
```

Comment:
Any reason why…
```diff
--- TextIOWrapper(BytesIO(X.encode("utf-8")))
+++ StringIO(X)
```
…would not work? (x4)

Reply:
Update: It would not, but there's an easy fix (please see https://github.com/pycontribs/ansi2html/pull/162/files#r800234179 above)

Reply:
No, that would not work. The tests then would fail in `ansi2html/converter.py:790` with an `AttributeError`

```
>       reader = io.TextIOWrapper(sys.stdin.buffer, opts.input_encoding, "replace")
E       AttributeError: '_io.StringIO' object has no attribute 'buffer'
```

Reply:
Yes. But if we use the old approach that got lost, StringIO works fine again, without buffer.

Reply:
Using the old method, mypy will complain about it:

```
ansi2html/converter.py:790: error: "TextIO" has no attribute "detach"
```